Makefile

```diff
@@ -71,3 +71,7 @@ vendor:
 	           -e GOFLAGS=-mod=mod \
 	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) go mod tidy -compat=$(GO_MOD_VERSION) && go mod vendor
 .PHONY: vendor
+
+check-uncommitted:
```

Comment:
Is this target intentionally manual?
It seems fit to be dependency of the `check` target.

Reply:
Nope, it's my bad..